wikibase/1.30/bundle/Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM ubuntu as fetcher
 
 RUN apt-get update && \
-    apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
+    apt-get install --yes --no-install-recommends unzip jq curl ca-certificates git
 ADD download-extension.sh .
 RUN bash download-extension.sh OAuth
 RUN tar xzf OAuth.tar.gz && rm *.tar.gz
+RUN git clone --depth 1 https://github.com/filbertkm/WikibaseImport.git
```

Comment:
Could also wget the master branch from github (but see my other comments about compatibility)